lib/march_hare/session.rb

```diff
@@ -608,6 +613,32 @@ def tls_certificate_password_from(opts)
       self.class.tls_certificate_password_from(opts)
     end
 
+    # @private
+    def init_default_exception_handler(logger)
+      StrictExceptionHandler.new(logger)
```

Comment:
I was never very happy about this default in the Java client, which goes back to 1.0 or so.

Reply:
I could use the other one(`ForgivingExceptionHandler`) as default if that's preferred I simply didn't want to change existing behavior

Reply:
That would be closer to how Bunny works.